shared/params/config.go

```diff
@@ -245,7 +245,7 @@ func MinimalSpecConfig() *BeaconChainConfig {
 	minimalConfig.MinPerEpochChurnLimit = 4
 	minimalConfig.ChurnLimitQuotient = 65536
 	minimalConfig.ShuffleRoundCount = 10
-	minimalConfig.MinGenesisActiveValidatorCount = 64
+	minimalConfig.MinGenesisActiveValidatorCount = 512
```

Comment:
Is this an intentional change?

Reply:
My bad, was using it for testing. I shouldn't have checked that in 😅